sources/mongodb/README.md

```diff
@@ -0,0 +1,54 @@
+# Mongo Database
+Mongo is a document-oriented database that store json-like documents is one of the most used NoSQL databases. The Mongo Database `dlt` is a verified source and pipeline example that makes it easy to load data from your Mongo database to a destination of your choice. It offers flexibility in terms of loading either the entire database or specific collections to the target.
+
+## Initialize the pipeline with Mongo Database verified source
+```bash
+dlt init mongodb bigquery
+```
+Here, we chose BigQuery as the destination. Alternatively, you can also choose redshift, duckdb, or any of the other [destinations.](https://dlthub.com/docs/dlt-ecosystem/destinations/)
+
+## Setup verified source
+
+To setup the Mongo Database Verified Source read the [full documentation here.](https://dlthub.com/docs/dlt-ecosystem/verified-sources/mongodb)
+
+## Add credentials
+1. Open `.dlt/secrets.toml`.
+2. In order to continue, you can use the a connection URL and database that looks like this:
+    ```bash
+    connection_url = "mongodb://dbuser:passwd@host.or.ip:27017"
+    database = "local"
+    ```
+    Here's what the `secrets.toml` looks like:
+    ```toml
+    # Put your secret values and credentials here. do not share this file and do not upload it to github.
+    # We will set up creds with the following connection URL, which is a public database
+    
+    # The credentials are as follows
+    connection_url = "mongodb://dbuser:passwd@host.or.ip:27017" # Driver name for the database
+    database = "local" # Database name
+    ```
+3. Enter credentials for your chosen destination as per the [docs.](https://dlthub.com/docs/dlt-ecosystem/destinations/)
+
+## Running the pipeline example
+
+1. Install the required dependencies by running the following command:
+    ```bash
+    pip install -r requirements.txt
+    ```
+    
+2. Now you can build the verified source by using the command:
+    ```bash
+    python3 mongodb_pipeline.py
+    ```
+    
+3. To ensure that everything loads as expected, use the command:
+    ```bash
+    dlt pipeline <pipeline_name> show
+    ```
+    
+    For example, the pipeline_name for the above pipeline example is `local_mongo`, you can use any custom name instead.
+    
+
```

Comment:
You could add another chapter ie
## How to use

and here you explain that by default we reflect all collections in database, we add primary keys and you can select the collections before loading as demonstrated in demo script

or you can load single collection by calling ... function

we'll improve this doc anyway but this is essential information